pkg/controller/utils/utils.go

```diff
@@ -71,3 +75,22 @@ func AddFinalizer(object metav1.Object, finalizer string) {
 	finalizers.Insert(finalizer)
 	object.SetFinalizers(finalizers.List())
 }
+
+// LogAwsError formats and logs aws error and returns if err was an awserr
+func LogAwsError(logger logr.Logger, errMsg string, returnError error, err error) bool {
+	if aerr, ok := err.(awserr.Error); ok {
+		if returnError == nil {
```

Comment:
Why is `returnError` passed in? It's being set to `aerr` which could be directly used in `logger.Error` and is not passed in via pointer so it won't propagate up when `LogAwsError` returns. Can you clarify what this is used for?

Reply:
This statement is so that if one would like to return an error that is neither the `err` or `aerr` in the call to `logger.Error` they may. For example if you would like change the value based on a switch in the proceeding code. 

Places where this occurs
- [L160](https://github.com/openshift/aws-account-operator/pull/194/files#diff-f899a0fa089ce8f0b7744fc82ee53558R160)
-[L927](https://github.com/openshift/aws-account-operator/pull/194/files#diff-
32f12988d73ded1726ea7c67c8cb1f90R927)
-[L859[](https://github.com/openshift/aws-account-operator/pull/194/files#diff-32f12988d73ded1726ea7c67c8cb1f90R895)